plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -417,6 +441,66 @@ private static LongReadFunction sqlServerDateReadFunction()
         return (resultSet, index) -> LocalDate.parse(resultSet.getString(index), DATE_FORMATTER).toEpochDay();
     }
 
+    private static ColumnMapping timestampWithTimeZoneColumnMapping(int precision)
+    {
+        checkArgument(precision <= MAX_DATETIMEOFFSET_PRECISION, "Unsupported datetimeoffset precision %s", precision);
```

Comment:
nit: Existing messages in other mappings is `Precision is out of range: %s`.

Can you change others to follow the same form as you did here?